models/ed/R/veg2model.ED2.R

```diff
@@ -18,10 +38,9 @@ veg2model.ED2 <- function(outfolder, start_date, end_date,
   formatnames <- c("ED2.cohort", "ED2.patch", "ED2.site")
   dbfilenames <- c("css.file", "pss.file", "site.file")
   
-  file.prefix <- paste("siteid", site_id, source, start_year, end_year,
+  file.prefix <- paste(site_name, source, start_year,
```

Comment:
got it, then we'll understand to which site the file belongs only from the folder name? (without actually looking into the files I mean)